dev-env

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
```

Comment:
I chose Bash for it's widespread presence and because I'm familiar with it, but we can add support for other shells as well.